### ISSUE TITLE
Option to have simplified, traditional, or both of the classifiers, and other fields.

### ISSUE THREAD

**Issue by gdelmee** [Gustaf-C/anki-chinese-support-3#15]
As of now, autofill will fulfil the classifier field with both traditional and simplified classifiers, even we have the option to only fetch traditional, simplified, of both hanzi.

![image](https://github.com/Gustaf-C/anki-chinese-support-3/assets/17324443/6dd0861a-dfad-4a4d-87a0-9d08e61b1ece)

In order to match and have consistency in behavior, it could be great to have an option to choose between those three options (global or only for the classifiers, as the split is already in place for hanzi, but could also be useful then for ruby, examples, etc.), or new fields to separate those.

Fields to separate can still be useful for people learning both traditional and simplified as you might want to put classifier in front of the ruby notation in the answer:

![image](https://github.com/Gustaf-C/anki-chinese-support-3/assets/17324443/cdc080bb-32dd-4578-bfbc-498b3a7319e8)

**Comment by Gustaf-C** [comment]
Yes this is something that has been bothering me for a while as well, thank you for registering it.
Unfortunately it is quite far down on the list of priorities atm.

**Comment by Gustaf-C** [comment]
Is this what you had in mind?

![image](https://github.com/Gustaf-C/anki-chinese-support-3/assets/106698658/7b38c475-4577-4953-a165-1909eb215dd0)

**Comment by gdelmee** [comment]
Indeed, adding new fields could be a way to keep the compatibility with previous versions/existing cards.

Persons being interested in simplified or traditional could then add the relevant field.
Also, wanting to move forward later on adding the other one and/or the combined one with bulk filling should be good.
